### PR TITLE
Rework stop button

### PR DESCRIPTION
**Pull request by cbocta**

### CODE REVIEW

**Comment by ssavrim** [comment]
:+1:

File: www/base/src/app/builders/builds/build.controller.coffee [hunk 1]
```diff
@@ -14,8 +14,6 @@ class Build extends Controller
             $scope.is_rebuilding = true
 
             success = (res) ->
-                console.log res
-                $scope.is_rebuilding = false
                 refreshContextMenu()
```
Comment (tardyp) [inline www/base/src/app/builders/builds/build.controller.coffee]: You don't need to refresh the context menu

File: www/base/src/app/builders/builds/build.controller.coffee [hunk 2]
```diff
@@ -33,7 +31,6 @@ class Build extends Controller
             $scope.is_stopping = true
 
             success = (res) ->
-                $scope.is_stopping = false
                 refreshContextMenu()
```
Comment (tardyp) [inline www/base/src/app/builders/builds/build.controller.coffee]: You don't need to refresh the context menu

**Comment by ssavrim** [comment]
:+1:

**Comment by sa2ajj** [comment]
(this is ui otherwise i'd merge it :))

**Comment by tardyp** [comment]
thanks

**Pull request merged**